main.go

```diff
@@ -395,6 +405,66 @@ func NewExporter(server string, timeout time.Duration, logger log.Logger) *Expor
 			[]string{"slab"},
 			nil,
 		),
+		itemsHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "hot_items"),
+			"Number of items presently stored in the HOT LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsWarm: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "warm_items"),
+			"Number of items presently stored in the WARM LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsCold: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "cold_items"),
+			"Number of items presently stored in the COLD LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsTemporary: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "temporary_items"),
+			"Number of items presently stored in the TEMPORARY LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsAgeOldestHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "hot_age_seconds"),
+			"Age of the oldest item in HOT LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsAgeOldestWarm: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "warm_age_seconds"),
+			"Age of the oldest item in HOT LRU.",
+			[]string{"slab"},
+			nil,
+		),
+		itemsHitsToHot: prometheus.NewDesc(
+			prometheus.BuildFQName(namespace, subsystemSlab, "items_hits_to_hot"),
```

Comment:
A couple more naming convention changes. All of the counters should end in `_total`. Maybe we can call these something like `hot_lru_hits_total`.

Reply:
Yep the name is definitely better, I was trying to keep compatibility with something like `items_moves_to_warm`. I can rename since these are new metrics, but some refactoring to the other ones at this point would be good. The only problem that I can see is breaking people already using the exporter..